dev/integration_tests/android_engine_test/android/app/src/main/kotlin/com/example/native_driver_test/MainActivity.kt

```diff
@@ -31,13 +32,24 @@ class MainActivity : FlutterActivity() {
                 add(NativeDriverSupportPlugin())
             }
 
+        flutterEngine
```

Comment:
This isn't great and we probably need to fix the registry so that it is shared across controllers.

Reply:
I have a follow up patch to fix this.

Reply:
Thank you!

Add a TODO/issue.